apps/package.json

```
@@ -11,6 +11,8 @@
     "lint": "yarn lint:scss && yarn lint:js",
     "lint:fix": "yarn lint:scss --fix && yarn lint:js --fix",
     "lint:js": "eslint --format ./.eslintCustomMessagesFormatter.js --ext=.js,.jsx,.ts,.tsx --report-unused-disable-directives-severity=error .",
```

For consistency between environments, what do you think about

```suggestion
    "lint:js": "concurrently \"yarn lint:js:ts\" \"yarn lint:jsx:tsx\""
```

And below, you would continue to call `yarn lint`

---

I like it! I've extracted this to a new PR so that it can be rolled back more easily if we have a problem with the `concurrently` package: https://github.com/code-dot-org/code-dot-org/pull/69087